src/main/resources/dev/definitions/prisons/orphanage/ors-presumptive-recategorisation-scheme-report.json

```diff
@@ -471,14 +471,15 @@
       },
       "specification": {
         "template": "list",
```

Comment:
Does the template need to be list-section instead of list when there is a section?